ntc_templates/templates/fortinet_get_system_interface.textfsm

```diff
@@ -17,6 +18,8 @@ Value NETMASK (\d+?\.\d+?\.\d+?\.\d+?)
 Value STATUS (\S+)
 Value NETBIOS_FORWARD (\S+)
 Value TYPE (\S+)
+Value RING_RX (\d+?)
+Value RING_TX (\d+?)
```

Comment:
Hello @yone2ks

1. This question isn't to say the ring data isn't useful (because I don't run fortinet gear). _Are the ring tx and rx counters valuable data to capture?_

1. Considering it is valuable to capture, since the rule makes the capture group optional there isn't a need to make it optional in the Value/capture group definition.

```suggestion
Value RING_RX (\d+)
Value RING_TX (\d+)
```